### ISSUE TITLE
MNDO99 and N2 

### ISSUE THREAD

**Issue by tobigithub** [qcxms/QCxMS#4]
Hi,
I was wondering what the correct settings for MNDO99 and N2 are? I used these, according to the [documentation](https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms_run.html):
````
>mndo
>cid
>iatom n2
>elab 40
>maxcoll 6
>noesi
````
Is that already implemented? I can run the MD steps and prep step, but not the actual production run it stops with:
````
Exit: CID with MNDO99 only with N2
````
Thanks
Tobias

**Comment by JayTheDog** [comment]
This was a bug where the N2 molecule was given the wrong index number. Fixed with the QCxMS 5.1.0 release. Thanks for bringing this up!

**Comment by tobigithub** [comment]
Thank you.
Tobias
